var/spack/repos/builtin/packages/py-sonata-network-reduction/package.py

```diff
@@ -13,6 +13,7 @@ class PySonataNetworkReduction(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/nse/sonata-network-reduction"
 
     version('develop', branch='master')
+    version('0.0.9', tag='sonata-network-reduction-v0.0.9')
```

Comment:
Do you plan on deploying this, too? I'm not seeing a change in `bbp-applications.yaml`.

Reply:
Thank you! I forgot.